src/containers/App.js

```diff
@@ -7,40 +7,39 @@ import LoginForm from './LoginForm';
 import MoviesDisplay from './MoviesDisplay';
 import Header from './Header';
 
-
 class App extends Component {
-constructor(props) {
-  super()
-}
+  constructor() {
+    super();
+  }
 
-componentDidMount = async () => {
-  const { handleFetch } = this.props;
-  fetchMovieData()
-  .then(data => data)
-  .then(movies => handleFetch(movies))
-}
+  componentDidMount = () => {
+    const { handleFetch } = this.props;
+    fetchMovieData()
+      .then(data => data)
+      .then(movies => handleFetch(movies));
+  };
 
   render() {
-    const homeDisplay = (
-      (this.props.movies.length) && 
-      <MoviesDisplay />
-    )
+    const homeDisplay = this.props.movies.length && <MoviesDisplay />;
     return (
-      <main className='app'>
+      <main className="app">
```

Comment:
are you a double quotes person?? I can become one if you need me to!

Reply:
Same, I typically go with singles but can switch to doubles for consistency.